app/src/main/java/org/xwiki/android/sync/auth/AuthenticatorActivity.kt

```diff
@@ -268,10 +268,10 @@ class AuthenticatorActivity : AccountAuthenticatorActivity() {
         } else {
             url += "/bin/view/XWiki/Registration"
         }
-        val intent = openLink(
-            url
-        )
-        startActivity(intent)
+
+        val regIntent = Intent(this, RegistrationActivity::class.java)
+        regIntent.putExtra("url", url)
```

Comment:
It is better to use some `const val` instead of hardcoding of variable like "url". We have file with constants (https://github.com/xwiki-contrib/android-authenticator/blob/master/app/src/main/java/org/xwiki/android/sync/Constants.kt) for this purpose

Reply:
This has been updated.